src/web/components/Core/DisplaySecret.scss

```diff
@@ -0,0 +1,11 @@
+.key-secret-reveal {
```

Comment:
this name should probably change to `.display-secret` since we have changed the name of the component + file